### DISCUSSION TITLE
How to use the plugin proxy-mirror correct

### DISCUSSION THREAD

**Discussion by arabot777** [apache/apisix#5439] [Q&A] [answered]
Now, I‘m running  ServiceA:8089 and ServiceA:8090 in local. I correctly configure the upstream and route of ServiceA:8089,and can be accessed normally via the curl domain.
Then I configure `proxy-mirror ` 's host is `http://127.0.0.1:8090` and  sample_ratio is `1`, later I curl ServiceA:8089 domain again, howerver  I cannot find any logs in ServiceA:8090.
Where is I negligent? Thanks!

**Comment by tokers** [comment]
How did you configure the `proxy-mirror` plugin, could you show us the configuration?

**Comment by arabot777** [comment, marked as answer]
sorry,lt's my fault.
I configure proxy-mirror is 
```
"proxy-mirror": {
      "disable": false,
      "host": "http://127.0.0.1:8090",
      "sample_ratio": 1
    }
```
but apisix service it not running in local. I should configure this way:
```
"proxy-mirror": {
      "disable": false,
      "host": "http://192.168.56.19:8090",
      "sample_ratio": 1
    }
```
`192.168.56.19` is my local ip. Now lt's working.
